db/migrations/versions/012_event_data_to_jsonb.py

```diff
@@ -22,12 +22,12 @@ def upgrade():
 
     # PROJECT PROGRESS #
     with op.batch_alter_table("project_progress", schema=None) as batch_op:
-        batch_op.add_column(sa.Column("event_data_blob", postgresql.JSONB(astext_type=sa.Text()), nullable=True))
+        batch_op.add_column(sa.Column("data_blob", postgresql.JSONB(astext_type=sa.Text()), nullable=True))
```

Comment:
Please dont alter historical (already applied migrations) as it won't reflect actual system changes and also its easy to get the dB in a bad state

Reply:
I think it requires a new migration yo rename the existing cols, sorry